pandas/core/ops/missing.py

```diff
@@ -88,8 +88,10 @@ def mask_zero_div_zero(x, y, result):
     Examples
     --------
     >>> x = np.array([1, 0, -1], dtype=np.int64)
+    >>> x
+    array([ 1,  0, -1])
     >>> y = 0       # int 0; numpy behavior is different with float
-    >>> result = x / y
+    >>> result = x // y
```

Comment:
comment in code: no need to do this for truediv; in py3 numpy behaves the way we want.

maybe update the docstring to reflect this change.

Reply:
@simonjayhawkins I'm not sure I understand what to update, can you be more specific?

Reply:
https://github.com/pandas-dev/pandas/blob/6f93898d32c0f1fdb382d1e9dee434c158998374/pandas/core/ops/missing.py#L73-L76